### PR TITLE
fix: platform script path works on windows

### PR DESCRIPTION
**Pull request by Letiste**
Fix https://github.com/flutter/flutter/issues/94285

Here is the explanation: https://github.com/dart-lang/sdk/issues/31610

### CODE REVIEW

File: dashboard/test/utils/task_icons.dart [hunk 1]
```diff
@@ -13,7 +13,7 @@ import 'package:path/path.dart' as path;
 Future<void> precacheTaskIcons(WidgetTester tester) async {
   // Depending on how we're invoked, Platform.script.path will have extra parts
   // after app_flutter. Just trim them off.
-  final List<String> pathParts = path.split(Platform.script.path);
+  final List<String> pathParts = path.split(Platform.script.toFilePath());
```
Comment (CaseyHillers) [inline dashboard/test/utils/task_icons.dart:16]: Will this work as expected?

Based on the the linked issue, it'll add file:// which is not what we want here. For context, this app is primarily deployed to the web, but we do try and support the other platforms when possible

Comment (Letiste) [reply dashboard/test/utils/task_icons.dart:16]: This is only used in the tests, before pumping any widgets. We need to get the correct file path to read the images (or at least that's my understanding, sorry if I'm mistaken)

Comment (CaseyHillers) [reply dashboard/test/utils/task_icons.dart:16]: It's using `ExactAssetImage` which does the resolution for the platform the app is running on. I'll close this PR as obsolete

**Comment by CaseyHillers** [comment]
Thank you for the PR! I'm closing this as obsolete, however, please reopen if I missed anything.

**Pull request closed without merging**